plugins/node/opentelemetry-instrumentation-winston/src/instrumentation.ts

```diff
@@ -39,10 +47,24 @@ import { VERSION } from './version';
 
 const winston3Versions = ['>=3 <4'];
 const winstonPre3Versions = ['>=1 <3'];
+const DEFAULT_CONFIG: WinstonInstrumentationConfig = {
+  disableLogSending: false,
+  disableLogCorrelation: false,
+};
 
 export class WinstonInstrumentation extends InstrumentationBase {
+  private _logger: Logger;
+
   constructor(config: WinstonInstrumentationConfig = {}) {
-    super('@opentelemetry/instrumentation-winston', VERSION, config);
+    super(
+      '@opentelemetry/instrumentation-winston',
+      VERSION,
+      Object.assign({}, DEFAULT_CONFIG, config)
+    );
+    this._logger = logs.getLogger(
```

Comment:
I wonder if this works with the example in readme.  sdk.start() sets the global log provider but this constructor is ran before it so this._logger will always be noop no?